azurerm/data_source_app_service_test.go

```diff
@@ -0,0 +1,199 @@
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAppService_siteConfig(rInt, location),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "site_config.0.use_32_bit_worker_process", "true"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceAzureRMAppService_appSettings(t *testing.T) {
+	dataSourceName := "data.azurerm_app_service.test"
+	rInt := acctest.RandInt()
+	location := testLocation()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAppService_appSettings(rInt, location),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "app_settings.foo", "bar"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceAzureRMAppService_connectionString(t *testing.T) {
+	dataSourceName := "data.azurerm_app_service.test"
+	rInt := acctest.RandInt()
+	location := testLocation()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAppService_connectionStrings(rInt, location),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "connection_string.0.name", "Example"),
+					resource.TestCheckResourceAttr(dataSourceName, "connection_string.0.value", "some-postgresql-connection-string"),
+					resource.TestCheckResourceAttr(dataSourceName, "connection_string.0.type", "PostgreSQL"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceAppService_basic(rInt int, location string) string {
+	config := testAccAzureRMAppService_basic(rInt, location)
+	return fmt.Sprintf(`
+%s
+
+data "azurerm_app_service" "test" {
+  name                = "${azurerm_app_service.test.name}"
+  resource_group_name = "${azurerm_app_service.test.resource_group_name}"
+}
+`, config)
+}
+
+func testAccDataSourceAppService_tags(rInt int, location string) string {
+	config := testAccAzureRMAppService_tags(rInt, location)
+	return fmt.Sprintf(`
+%s
+
+data "azurerm_app_service" "test" {
+  name                = "${azurerm_app_service.test.name}"
+  resource_group_name = "${azurerm_app_service.test.resource_group_name}"
+}
+`, config)
+}
+
+func testAccDataSourceAppService_clientAffinityDisabled(rInt int, location string) string {
+	config := testAccAzureRMAppService_clientAffinityDisabled(rInt, location)
+	return fmt.Sprintf(`
+%s
+
+data "azurerm_app_service" "test" {
+  name                = "${azurerm_app_service.test.name}"
+  resource_group_name = "${azurerm_app_service.test.resource_group_name}"
+}
+`, config)
+}
+
+func testAccDataSourceAppService_siteConfig(rInt int, location string) string {
+	config := testAccAzureRMAppService_32Bit(rInt, location)
```

Comment:
Is there a reason for the change of the test name from `testAccAzureRMAppService_32Bit` -> `testAccDataSourceAppService_siteConfig`?